features/auditing.rst

```diff
@@ -69,31 +69,85 @@ The wildcard (\*) can be used to expand matching in different ways, e.g.:
 The Method property is optional. If left out, null, empty or given the value \*, it will match all HTTP verbs. 
 You can enter multiple verbs, delimited by the \| symbol (e.g. GET\|POST).
 
+.. _configure_audit_log_file:
+
 Audit log file configuration
```

Comment:
Please add a paragraph on sync/async and the tradeoff between performance and reliability.

Reply:
Done!